File: src/easy/q9_PalindromeNumber/Solution.java
package easy.q9_PalindromeNumber;

class Solution {
    public boolean isPalindrome(int x) {
        String number = String.valueOf(x);
        int length = number.length();
        for (int i = 0; i < length / 2; i++) {
            if (number.charAt(i) != number.charAt(length - i - 1)) {
                return false;
            }
        }

        return true;
    }
}

File: src/easy/q9_PalindromeNumber/Solution.js
/**
 * @param {number} x
 * @return {boolean}
 */
let isPalindrome = function(x) {
    num = x.toString().split("");
    for (let i = 0; i < num.length / 2; i++) {
        if (num[i] !== num[num.length - i - 1]) {
            return false;
        }
    }
    return true;
};